### PR TITLE
admin_guide/manage_scc: couple of improvements

### PR DESCRIPTION
**Pull request by php-coder**
PTAL @pweil-

### CODE REVIEW

**Comment by php-coder** [comment]
Ping @pweil-

**Comment by php-coder** [comment]
@pweil- @mfojtik PTAL.

File: admin_guide/manage_scc.adoc [hunk 1]
```diff
@@ -243,7 +241,7 @@ The following describe common scenarios and procedures using SCCs.
 === Grant Access to the Privileged SCC
 
 In some cases, an administrator might want to allow users or groups outside the
-administrator group access to create more privileged pods. To do so, you can:
+administrator group access to create more **privileged pods**. To do so, you can:
```
Comment (ahardin-rh) [inline admin_guide/manage_scc.adoc]: To provide emphasis use this format instead:

    _privileged pods_

File: admin_guide/manage_scc.adoc [hunk 2]
```diff
@@ -260,6 +258,13 @@ For example, to allow the *e2e-user* access to the *privileged* SCC, run:
 $ oadm policy add-scc-to-user privileged e2e-user
 ----
 
+[WARNING]
+====
+Granting access to a user works only for the pods. For other resources
```
Comment (ahardin-rh) [inline admin_guide/manage_scc.adoc]: You can only grant access to a user for the pods. For other resources like deployments and StatefulSets, give access to a service account under which the related controller is operated upon.

File: admin_guide/manage_scc.adoc [hunk 3]
```diff
@@ -277,6 +282,10 @@ Then, add the service account to the `privileged` SCC.
 $ oadm policy add-scc-to-user privileged system:serviceaccount:myproject:mysvcacct
 ----
 
+The next step is to ensure that resource is being created in behalf of the
```
Comment (ahardin-rh) [inline admin_guide/manage_scc.adoc]: Then, ensure that the resource is being created on behalf of the service account. To do so, set the . . .

**Comment by ahardin-rh** [comment]
@php-coder Just a few minor comments from me. This will also need to be squashed into a single commit before we can merge. Perhaps @pweil- has other comments, however. Thanks!

File: admin_guide/manage_scc.adoc [hunk 4]
```diff
@@ -183,9 +183,7 @@ $ oc delete scc <scc_name>
 
 [NOTE]
 ====
-If you delete the default SCCs, they will not be regenerated upon restart,
-unless you delete all SCCs. If any constraint already exists within the system,
-no regeneration will take place.
+If you delete the default SCC, it will be regenerated upon restart.
```
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: s/the/a

File: admin_guide/manage_scc.adoc (hunk 2 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: Granting access to a user only works when the user directly creates a pod.  For pods created on behalf of a user, in most cases by the system itself, access ....

Examples of resources that create pods on behalf of a user are ...

**Review by pweil- — CHANGES_REQUESTED** [review]

**Comment by php-coder** [comment]
@ahardin-rh @pweil- Thank you for the comments! I've updated the PR and incorporated all these comments. PTAL.

File: admin_guide/manage_scc.adoc [hunk 5]
```diff
@@ -277,6 +284,10 @@ Then, add the service account to the `privileged` SCC.
 $ oadm policy add-scc-to-user privileged system:serviceaccount:myproject:mysvcacct
 ----
 
+Then, ensure that the resource is being created on behalf of the service
+account. To do so, set the `spec.serviceAccountName` field to a service account
```
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: Leaving the service account name blank will result in the `default` service account being used.

**Review by pweil- — APPROVED** [review]
minor nit and LGTM.  thanks!

File: admin_guide/manage_scc.adoc (hunk 5 above)
Comment (php-coder) [inline admin_guide/manage_scc.adoc]: Good addition! Updated.

**Comment by ahardin-rh** [comment]
Thanks so much!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/manage_scc.adoc#admin-guide-manage-scc[Managing Security Context Constraints]
|Added a warning about pod creation on behalf of a user within the xref:../admin_guide/manage_scc.adoc#grant-access-to-the-privileged-scc[Grant Access to the Privileged SCC] section.
%

**Pull request merged**